package Exception;

public class Exception1 {
	void activity() throws Exception
	{
		System.out.println("Inside Program 1 activity");
	}
	private void sout() {
		// TODO Auto-generated method stub

	}

}
